docker-compose.yml

```diff
@@ -29,17 +27,33 @@ services:
     expose:
       - 5000
     ports:
-      - 5000:5000
-    command: ["postgraphile", "--connection", "postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DATABASE}", "--host", "0.0.0.0", "--port", "5000", "--schema", "trials_status_schema"]
+      - "${HOST_IP:-127.0.0.1}:${GRAPHQL_PORT}:${GRAPHQL_PORT}"
```

Comment:
Is the curly brace maybe off here? `${HOST_IP}:`

Reply:
Also, wasn't sure about the negative sign here `-127.0.0.1` but that could easily just be some convention I don't know

Reply:
Yeah! It looks odd for sure. This is the bash syntax for reading an environment variable with default: https://stackoverflow.com/a/2013589/3376921 it works for docker compose as well.